packages/vue/src/utilities/directives/focus-directive.js

```diff
@@ -0,0 +1,27 @@
+function mousedownHandler(el) {
+  el.style.outline = "";
+}
+
+function keydownHandler(el) {
+  el.blur();
+}
+
+export const focus = {
+  inserted: function(el) {
+    el.tabindex = "0";
+  },
+  bind: function(el) {
+    el.addEventListener("mousedown", function() {
+      el.style.outline = "0";
```

Comment:
The default value for disabling `outline` is `none`.
```suggestion
      el.style.outline = "none";
```